gatsby-node.js

```diff
@@ -8,229 +8,172 @@
 
 const path = require('path')
 const fs = require('fs')
-const template = require('lodash/template')
-const wrap = require('lodash/wrap')
-const { defaultLocale, availableLocales } = require('./intl/config')
 const ServiceWorkerWebpackPlugin = require('serviceworker-webpack-plugin')
-const SvgStorePlugin = require('external-svg-sprite-loader/lib/SvgStorePlugin')
-
-module.exports.modifyWebpackConfig = ({ config, program, stage }) => {
-  // Allow requires from the src/ folder
-  config.merge({
-    resolve: {
-      root: path.join(__dirname, 'src')
-    },
-    target: 'web'
-  })
+const SvgStorePlugin = require('external-svg-sprite-loader')
+const { defaultLocale, availableLocales } = require('./intl/config')
+
+module.exports.onCreateWebpackConfig = ({ actions, getConfig, stage }) => {
+  let config = getConfig()
 
   // Fix Gatsby setting `resolve.modulesDirectories` to `path.resolve(__dirname, "node_modules")`
```

Comment:
change `resolve.modulesDirectories` to `resolve.modules`
change ``path.resolve(__dirname, "node_modules")` to just `node_modules`

Reply:
Sorry, missed that detail.